ntoskrnl/ke/i386/thrdini.c

```diff
@@ -485,6 +485,9 @@ KiDispatchInterrupt(VOID)
     }
     else if (Prcb->NextThread)
     {
+        /* Acquire the PRCB lock */
```

Comment:
For future reference: this comment is not useful. What _would_ be useful would be to mention that it's released by `KxQueueReadyThread`, since that's the part that's not obvious.